examples/README.md

```diff
@@ -194,30 +194,38 @@ Sources [ScalaRiakParquetExample.scala](./src/main/scala/com/basho/riak/spark/ex
 
 ```
 path/to/kafka/bin/zookeeper-server-start.sh config/zookeeper.properties
 path/to/kafka/bin/kafka-server-start.sh config/server.properties
-path/to/kafka/bin/kafka-topics.sh --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic streaming
 ```
 
 We also assume Riak KV/TS is installed and there is a Riak KV/TS node running at `127.0.0.1:8087`. You can find instruction to do so [here](http://docs.basho.com/riak/ts/latest/installing/). 
 
 Riak KV, Kafka and Spark master hostnames must be specified in [config.sh](./src/main/repl/conf/config.sh) prior to running the examples.
 
+**Important Note**: [config.sh](./src/main/repl/conf/config.sh) will attempt to gather your local scala version. This version number will be used in [run-example](./src/main/repl/bin/run-example) to pull the appropriate `spark-streaming-kafka` and `kafka` libraries from spark-packages.org. This can cause an error if your local Spark and the Spark-Riak-Connector were built for a different Scala version than your local Scala. For example, Spark 1.6.2 builds for Scala 2.10 even if you have Scala 2.11, and the Spark-Riak-Connector, by default, builds for Scala 2.10 as well. Therefore, in order for the streaming examples to work, the Scala version [config.sh](./src/main/repl/conf/config.sh) picks up must be the same as the Scala version that Spark and the Spark-Riak-Connector were built for. If you local Scala version is different than the version that Spark and the Spark-Riak-Connector were built for, you should change the Scala version in [this line](src/main/repl/conf/config.sh#L53) in [config.sh](./src/main/repl/conf/config.sh) to the version of Scala that Spark and the Spark-Riak-Connector were built for.
+
+
 ### Spark Streaming Riak KV Example
 
-This example will start a stream from the Kafka topic `streaming` into the KV bucket `test-data`. This stream will run until terminated. Whenever a message is produced for Kafka topic `streaming`, the Spark Streaming context will automatically stream the message from the topic into the KV bucket. To see this in action, we first need to run the example:
+This example will start a stream from the Kafka topic `ingest-kv` into the KV bucket `test-data`. This stream will run until terminated. Whenever a message is produced for Kafka topic `ingest-kv`, the Spark Streaming context will automatically stream the message from the topic into the KV bucket. To see this in action, we first need create the `ingest-kv` topic:
+
+```
+path/to/kafka/bin/kafka-topics.sh --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic ingest-kv
+```
+
+Then, start the example:
 
 ```
 /path/to/spark-riak-connector-examples/bin/run-example streaming.StreamingKVExample
```

Comment:
Please shorten `/path/to/spark-riak-connector-examples/bin/run-example` to `bin/run-example` so it matches the other examples, which assume that before running the examples users have already changed directories into `examples/src/main/repl`. It makes sense to keep the zookeeper and kafka paths as-is since they could be installed anywhere in a user's local env.